deepfence_server/pkg/integration/s3/s3.go

```diff
@@ -63,14 +63,18 @@ func (s S3) SendNotification(ctx context.Context, message string, extras map[str
 		fmt.Println("Failed to marshal JSON data", err)
 		return err
 	}
-
+	buf := &bytes.Buffer{}
+	gzWriter := gzip.NewWriter(buf)
```

Comment:
Here we can look into the `NewWriterLevel` where we can specify the compression level (which can be configurable).
With the `NewWriter`, we will always use the default compression.